cmd/kots/cli/reset-password.go

```diff
@@ -46,10 +46,14 @@ func ResetPasswordCmd() *cobra.Command {
 				os.Exit(1)
 			}
 
-			log.ActionWithoutSpinner("Reset the admin console password for %s", namespace)
-			newPassword, err := promptForNewPassword()
-			if err != nil {
-				os.Exit(1)
+			newPassword := v.GetString("new-password")
+			if newPassword == "" {
+				log.ActionWithoutSpinner("Reset the admin console password for %s", namespace)
```

Comment:
@salahalsaleh I think this sentence should be worded clearer. How about something like "The password for the Admin Console has been reset"...